PLOP.md

```diff
@@ -0,0 +1,38 @@
+#Plop
+
+Plop is a small component generator that can be used to quickly set up our web-components from pre designed templates.
+
+### Usage
+##### Component
+To use plop to generate a component follow these steps:
+
+````
+1. $ npm run generate component
```

Comment:
We are favouring "yarn" in this project

Reply:
Changed this to 'yarn generate component'